meteor/client/ui/Shelf/EndWordsPanel.tsx

```diff
@@ -73,10 +73,56 @@ export class EndWordsPanelInner extends MeteorReactComponent<
 
 export const EndWordsPanel = translateWithTracker<IEndsWordsPanelProps, IState, IEndsWordsPanelTrackedProps>(
 	(props: IEndsWordsPanelProps) => {
-		const { activePieceInstance } = getIsFilterActive(props.playlist, props.showStyleBase, props.panel)
-		return { livePieceInstance: activePieceInstance }
+		return { livePieceInstance: getPieceWithManus(props) }
 	},
 	(_data, props: IEndsWordsPanelProps, nextProps: IEndsWordsPanelProps) => {
 		return !_.isEqual(props, nextProps)
 	}
 )(EndWordsPanelInner)
+
+function getPieceWithManus(props: IEndsWordsPanelProps): PieceInstance | undefined {
+	const currentPartInstanceId: any = props.playlist.currentPartInstanceId
+
+	const unfinishedPiecesIncludingFinishedPiecesWhereEndTimeHaveNotBeenSet = getUnfinishedPieceInstancesReactive(
+		props.playlist,
+		props.showStyleBase
+	)
+	let highestStartedPlayback: Time = 0
+	const startedPiecesCount: number = 0
+	unfinishedPiecesIncludingFinishedPiecesWhereEndTimeHaveNotBeenSet.forEach((pieceInstance: PieceInstance) => {
+		if (pieceInstance.startedPlayback && pieceInstance.startedPlayback > highestStartedPlayback) {
+			highestStartedPlayback = pieceInstance.startedPlayback
+		}
+	})
+
+	const unfinishedPieces = unfinishedPiecesIncludingFinishedPiecesWhereEndTimeHaveNotBeenSet.filter(
```

Comment:
If `startedPiecesCount` is a dead variable, then the filter body can be simplified a bit :)